src/digitalIdentity-backend/src/main/resources/application.properties

```diff
@@ -15,4 +15,8 @@ spring.mail.port=587
 spring.mail.username=<to be filled>
 spring.mail.password=<to be filled>
 spring.mail.properties.mail.smtp.auth=true
-spring.mail.properties.mail.smtp.starttls.enable=true
\ No newline at end of file
+spring.mail.properties.mail.smtp.starttls.enable=true
+
+# Authentification:
+lissi.client.id= <to be filled>
```

Comment:
where do I get such a Authentification? A short hint in the Readme would be great :)